doc/research/05_acting.md

```diff
@@ -0,0 +1,199 @@
+# Basic research acting
+
+**Summary:** On this page you can find the results of the basic research on acting.
+
+---
+
+### Authors
+
+Gabriel Schwald, Julian Graf
+
+### Date
+
+14.11.2022
+
+---
+[[TOC]]
+
+## Objective
+
+The job of this domain is to translate a preplanned trajectory into actual steering controls for the vehicle.
+
+- safety:
+  - never exceeding vehicle limits
+  - never exceeding speed limits
+  - never leaf path
+- driving comfort?
+
+## Solutions from old PAF projects
+
+### [Paf 20/1](https://github.com/ll7/psaf2/wiki/Path-Tracking-Algorithmen)
+
+- modified [carla_ackermann_control](https://carla.readthedocs.io/projects/ros-bridge/en/latest/carla_ackermann_control/)
+- input: [twist-msgs](http://docs.ros.org/en/noetic/api/geometry_msgs/html/msg/Twist.html) (for velocity)
+- velocity control: PID
+- lateral control: PD (heading error)
+
+### [Paf 21/1](https://github.com/ll7/paf21-1/wiki/Vehicle-Controller)
+
+- input: waypoints
+- curve detection: returns distance to next curve
+- calculation of max curve speed as sqrt(friction_coefficient  *gravity_accel* radius)
+- in Curve: naive Controller
+- on straights: stanley controller
+- interface to rosbridge
+
+### [Paf 20/2](https://github.com/ll7/psaf2) and [Paf 21/2](https://github.com/ll7/paf21-2/tree/main/paf_ros/paf_actor#readme)
+
+- input: odometry(position and velocity with uncertainty), local path
+- lateral: Stanley Controller
+- speed controller: pid
+- ACC: (speed, distance) -> PID
```

Comment:
wofür steht ACC?

Reply:
Fixed in fd9e0bc59df8f9825da05b888d6c2d2cd19fb2e4